### PR TITLE
Revert "remove erroring tests"

### PR DESCRIPTION
**Pull request by czue**
Reverts dimagi/commcare-hq#14804

@nickpell I don't like the idea of these being permanently not run on travis. were you planning to look into why they broke and get them running again?

### CODE REVIEW

**Comment by czue** [comment]
actually these were failing locally for me so i just fixed them.

File: custom/enikshay/nikshay_datamigration/factory.py [hunk 1]
```diff
@@ -306,7 +321,11 @@ def get_drtb_hiv_referral_case_structure(self, episode_structure):
                 related_type=EPISODE_CASE_TYPE,
             )],
         }
-
+        if self.existing_drtb_hiv_case:
```
Comment (czue) [inline custom/enikshay/nikshay_datamigration/factory.py]: @nickpell we need to do this for any case being created by migration. was a bit surprised you hadn't already accounted for this given the similarities to other case types

Comment (czue) [inline custom/enikshay/nikshay_datamigration/factory.py]: would be good to add a test case for this too if not too hard

Comment (nickpell) [inline custom/enikshay/nikshay_datamigration/factory.py]: https://github.com/dimagi/commcare-hq/pull/14979

**Pull request merged**